data/json/mapgen/bike_shop.json

```diff
@@ -0,0 +1,159 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "s_bike_shop" ],
+    "object": {
+      "fill_ter": "t_grass",
```

Comment:
If you don't need to fill, it's better if you don't. That way the mapgen entry can be modernized easier in the future.

Reply:
Done.